src/base/system_parameter.c

```diff
@@ -2398,6 +2400,18 @@ static unsigned int prm_scal_perf_ps_repl_thread_ignore_unfix_flag = 0;
 static const bool prm_scal_perf_ps_repl_thread_ignore_unfix_default = false;
 bool PRM_SCAL_PERF_PS_REPL_THREAD_IGNORE_UNFIX_VALUE = prm_scal_perf_ps_repl_thread_ignore_unfix_default;
 
+static unsigned int prm_scal_perf_ps_req_responder_thread_count_flag = 0;
+static int prm_scal_perf_ps_req_responder_thread_count_default = 0;
+int PRM_SCAL_PERF_PS_REQ_RESPONDER_THREAD_COUNT_CURRENT_VALUE = 0;
+static int prm_scal_perf_ps_req_responder_thread_count_upper_value = 64;
+static int prm_scal_perf_ps_req_responder_thread_count_lower_value = 0;
```

Comment:
I think we can do it like the `prm_thread_core_count_default` does. 
```suggestion
static unsigned int prm_scal_perf_ps_req_responder_thread_count_flag = 0;
#if defined (SERVER_MODE)
static int prm_scal_perf_ps_req_responder_thread_count_default = (int) cubthread::system_core_count ();
#else
static int prm_scal_perf_ps_req_responder_thread_count_default = 1;
#endif
int PRM_SCAL_PERF_PS_REQ_RESPONDER_THREAD_COUNT_CURRENT_VALUE = 0;
static int prm_scal_perf_ps_req_responder_thread_count_upper_value = 64;
static int prm_scal_perf_ps_req_responder_thread_count_lower_value = 0;
```

With this, I think we can remove the setting default value in `get_maxim_extra_thread_count_by_server_type` and `server_request_responder()` like
```
   const size_t THREAD_COUNT =
 	  (size_t) ((prm_thread_count <= 0) ? std::thread::hardware_concurrency () : prm_thread_count);
   const size_t TASK_MAX_COUNT =
 	  (size_t) ((prm_task_count <= 0) ? (std::thread::hardware_concurrency () * 4) : prm_task_count);
```

Reply:
Done